democracy/models/section.py

```diff
@@ -49,6 +49,8 @@ class Section(Commentable, StringIdBaseModel):
     content = models.TextField(verbose_name=_('content'), blank=True)
     plugin_identifier = models.CharField(verbose_name=_('plugin identifier'), blank=True, max_length=255)
     plugin_data = models.TextField(verbose_name=_('plugin data'), blank=True)
+    plugin_source = models.URLField(verbose_name=_('plugin source code'), blank=True)
```

Comment:
Hm, I would keep the plugin_identifier since we probably will have non-map-questionnaires that want completely different data in the future. This way, we can tell the frontend to use a different React component when there is a need for that.